packages/sdk-js/src/mongrule.ts

```diff
@@ -15,26 +16,27 @@ const _regexCache: { [key: string]: RegExp } = {};
 // The top-level condition evaluation function
 export function evalCondition(
   obj: TestedObj,
-  condition: ConditionInterface
+  condition: ConditionInterface,
+  idLists: IdLists
```

Comment:
Even though this is marked as required here, it's worth doing a line at the top of this function: `idLists = idLists || {}`.

Since this function is being exported, others might be relying on the existing behavior and not everyone is using Typescript or even if they are, they might have a `@ts-ignore` comment or something here.